Tests/EndToEndTests/RepeatingEndToEndTests.swift

```diff
@@ -300,6 +300,42 @@ extension RepeatingEndToEndTests {
 
 // MARK: -
 
+fileprivate struct Foozle: ParsableArguments {
+  @Flag() var verbose: Bool
+  @Argument(parsing: .unconditionalRemaining) var names: [String]
+}
+
+extension RepeatingEndToEndTests {
+  func testParsing_repeatingUnconditionalArgument() throws {
+    AssertParse(Foozle.self, []) { foozle in
+      XCTAssertFalse(foozle.verbose)
+      XCTAssertEqual(foozle.names, [])
+    }
+    
+    AssertParse(Foozle.self, ["--other"]) { foozle in
+      XCTAssertFalse(foozle.verbose)
+      XCTAssertEqual(foozle.names, ["--other"])
+    }
+    
+    AssertParse(Foozle.self, ["--verbose", "one", "two", "three"]) { foozle in
+      XCTAssertTrue(foozle.verbose)
+      XCTAssertEqual(foozle.names, ["one", "two", "three"])
+    }
+    
+    AssertParse(Foozle.self, ["one", "two", "three", "--other", "--verbose"]) { foozle in
+      XCTAssertTrue(foozle.verbose)
+      XCTAssertEqual(foozle.names, ["one", "two", "three", "--other"])
+    }
+    
+    AssertParse(Foozle.self, ["--verbose", "--other", "one", "two", "three"]) { foozle in
+      XCTAssertTrue(foozle.verbose)
+      XCTAssertEqual(foozle.names, ["--other", "one", "two", "three"])
```

Comment:
Would be good to add a test case that include `--`, e.g.
```swift
AssertParse(Foozle.self, ["--verbose", "one", "--", "--other", "two", "three"]) { foozle in
```
in order to make sure we don’t change/regress the `--` terminator behaviour with `.unconditionalRemaining`.